papis/tui/widgets/diff.py

```diff
@@ -190,7 +192,7 @@ def oset(event: Event, option: str, value: bool) -> None:
 
     diffshow(
         texta=texta, textb=textb,
-        title="GENERAL DIFFERENCE",
+        title="all changes",
```

Comment:
```suggestion
        title="All changes",
```
? It would show up as `--- Diff view: All changes ---`. Not sure which way's better, but it popped out to me that it wasn't capitalized..

Reply:
The rule I go with is that after a colon we should capitalise the next word only when 2 or more full sentences follow (see [here](https://www.grammarbook.com/blog/commas/capitalization-with-colons/) for example), but I definitely don't have strong feelings about it haha

Reply:
That's good to know, thanks for the link! Then yes, it's just fine as is.

For my curiosity, does that also work for title capitalization? (I had in mind something like [this](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case), from a quick search)

Reply:
Good point, it could be considered a title. In that case I guess the question is whether we use (British style) sentence case for them (in which case my initial form would still be right) or we use (US style) title case, in which case we should capitalise even more (e.g. "Diff View: All Changes").

Reply:
:exploding_head: Yeah, it's completely fine for now, haha! Thank you for explaining.